WTWSMS/history/provinces/788 - Farama.txt

```diff
@@ -16,11 +16,26 @@ b_seyan = city
 #b_mustabiq = castle
 
 # Misc
-culture = greek
-religion = docetic 
+culture = coptic
+religion = sethian
```

Comment:
All the things I saw about the Sethians said that they were primarily concentrated in remote parts of Egypt. The first source listed below is the most definitive about large hordes of Sethians hiding in remote parts of Egypt.

http://jdt.unl.edu/lithist.html#Heading14 (section "Conclusion")

https://mandoxegypt.wordpress.com/2016/11/20/gnostic-sethians-sethian-gnostics/

http://www.jwmt.org/v3n24/rivera.html

http://www.bibliotecapleyades.net/sumer_anunnaki/reptiles/reptiles62.htm